test/helper.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
```

Comment:
Can we name this file `certificates.js` instead of `helper.js`?

Reply:
Not happy with that. If I see a file `test/certificates.js` I would expect it contains tests for certificates, not helper functions for certificates. Also, `test/helper.js` will contains future test needs.

Reply:
Not happy with that. If I see a file test/certificates.js I would expect it contains tests for certificates, not helper functions for certificates. Also, test/helper.js will contains future test needs.

Reply:
Tests would be `something.test.js`